lib/Transforms/Instrumentation/ShadowMemSeaDsa.cc

```diff
@@ -478,6 +491,20 @@ class ShadowDsaImpl : public InstVisitor<ShadowDsaImpl> {
     return *ci;
   }
 
+  CallInst &mkArgNewModNode(IRBuilder<> &B, Constant *argFn, const dsa::Cell &c,
```

Comment:
We decided to revert this, to not include extra arguments in `shadow.mem` calls

Reply:
This was needed for https://github.com/seahorn/seahorn/pull/234#discussion_r338634001